app/controllers/pet_applications_controller.rb

```diff
@@ -4,6 +4,10 @@ def new
     @favorite_pets = favorite.load_favorite_pets
   end
 
+  def show
+    @pet_application = PetApplication.find(params[:id])
+  end
+
   def create
     new_application = PetApplication.new(application_params)
```

Comment:
Pretty sure this is the call in the test suite to hard make an application... & you can just serve up the expected applicaton_params in the test setup...